src/App.js

```diff
@@ -271,13 +271,6 @@ const returnDefaultComponent = (Component, props) => {
           <Navigation {...props}/>
           <Component {...props} />
           <RequestErrorModal {...props} />
```

Comment:
I agree we should change this modal as well, but I would suggest doing this in a separate PR as the refactor is a little more complex than it looks since the `RequestErrorModal` is displayed from inside a `axiosInstance` which has no access to the `GlobalModalContext`. We will probably have to create some mechanism of displaying the `GlobalModal` from a redux action

Also, it is not affected by the same problem as it is rendered on `App.js`

Reply:
Ok. Do we have any other component in this situation? And do we have a KTLO/issue to handle this?